frontend/SettingsPane.css

```diff
@@ -0,0 +1,87 @@
+.SettingsPane {
+  display: flex;
+  flex-wrap: wrap;
+  flex-shrink: 0;
+  align-items: center;
+  position: relative;
+  background-color: var(--base01);
+  border-bottom: 1px solid var(--base03);
```

Comment:
Should these vars use a `theme` prefix (e.g. `--theme-based01` or `--theme-base-01`)?